security/pkg/workload/server.go

```diff
@@ -69,11 +73,20 @@ func (s *SDSServer) GetTLSCertificate(udsPath string) *auth.TlsCertificate {
 // FetchSecrets fetches the X.509 key/cert for a given workload whose identity
 // can be derived from the UDS path where this call is received.
 func (s *SDSServer) FetchSecrets(ctx context.Context, request *api.DiscoveryRequest) (*api.DiscoveryResponse, error) {
+	// Get the uds path where this call is received.
+	md, ok := metadata.FromIncomingContext(ctx)
+	if !ok || len(md[udsPathKey]) == 0 {
+		errMessage := "Unknown UDS path."
```

Comment:
s/Unknown/unknown 

Typically the error message starts with the first word in small case.

Reply:
done